packages/mui-base/src/ModalUnstyled/ModalUnstyled.types.ts

```diff
@@ -123,6 +123,10 @@ export interface ModalUnstyledOwnProps {
    * If `true`, the component is shown.
    */
   open: boolean;
+  /**
+   * If `true`, the exit transition is completed
+   */
+  exited: boolean;
```

Comment:
The Modal doesn't use such a prop. Why is it needed here?

Reply:
Sorry misinterpreted a Typescript error message.